docs/src/api/class-locatorassertions.md

```diff
@@ -1236,7 +1236,7 @@ await Expect(locator).toHaveAccessibleNameAsync("Save to disk");
 
 ### param: LocatorAssertions.toHaveAccessibleName.name
 * since: v1.44
-- `name` <[string]|[RegExp]>
+- `name` <[string]|[RegExp]|[Array]<[string]>|[Array]<[RegExp]>>
```

Comment:
This should be [Array]<[string]|[RegExp]>

Reply:
Fixed the type format